client/src/components/Search.jsx

```diff
@@ -199,15 +203,21 @@ const Search = (props) => {
     setCoopSearchSettings({...coopSearchSettings, [name]: selections})
   }
 
+  const handleToggle = (bool) => setListView(bool)
   // same logic from Search.jsx
   const renderSearchResults = () => {
+    console.log("is auth: " + isAuthenticated);
```

Comment:
If you feel this is in a good place, let's remove the console.logs

Reply:
My bad. I'll definitely remove this.